lib/formatters/__tests__/verboseFormatter.test.js

```diff
@@ -261,4 +261,74 @@ path/to/file.css
 0 problems found`.trimStart(),
 		);
 	});
+
+	it('outputs input CSS', () => {
+		const results = [
+			{
+				source: '<input css>',
+				errored: true,
+				warnings: [
+					{
+						line: 1,
+						column: 2,
+						rule: 'bar',
+						severity: 'error',
+						text: 'Unexpected foo',
+					},
+				],
+				deprecations: [],
+				invalidOptionWarnings: [],
+			},
+		];
+
+		const output = prepareFormatterOutput(results, verboseFormatter);
+
+		expect(output).toBe(
```

Comment:
Minor nit, can we use `const { stripIndent } = require('common-tags');` here and below to be consistent with how we mostly do it else where?

Reply:
Good idea! Fixed via e8429207.

I also fixed the existing test cases in the module for consistency.